### PR TITLE
feat: ability to run amplify command to generate code files

### LINKED ISSUE #278: Improve Amplify Code Generation
Introduced in PR #269, The react app is leveraging GraphQL for the API. As part of this, Amplify codegen will create the Mutations, Query & Subscription TypeScript Objects, as well as the API object. 

With the changes, the generated files are included, but there's not an easy-to-update way to change the generated code without deploying the back-end first. 

It would be preferred if an `npm run` command or something similar could be added to improve the developer experience when updating the API

### PR DESCRIPTION
**Pull request by Rob-Powell**
*Issue #, if available:* #278

*Description of changes:*
This resolves #278 by implemented a dev command `npm run gen` which runs the amplify cli to generate files based off schema.graphql reducing developer burden with appsync solution.

https://docs.amplify.aws/javascript/build-a-backend/graphqlapi/client-code-generation/

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by bigadsoleiman — APPROVED** [review]

File: .graphqlconfig.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+projects:
+  test:
```
Comment (massi-ang) [inline .graphqlconfig.yml:2]: maybe change the name of the project to `chatbot`

**Comment by massi-ang** [comment]
Thanks for this contribution. Please remove the `schema.json` file from the `lib/user-interface/react-app` since it is not used and will never be refreshed.

**Review by massi-ang — CHANGES_REQUESTED** [review]
This change makes the `lib/user-interface/react-app/schema.json` redundant. Please remove.

File: .graphqlconfig.yml (hunk 1 above)
Comment (Rob-Powell) [reply .graphqlconfig.yml]: updated

**Review by massi-ang — APPROVED** [review]

**Pull request merged**